rdbi-recipes/pom.xml

```diff
@@ -89,6 +89,11 @@
             <artifactId>awaitility</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
scope should be test only, but i'm surprised this has to be added. is this correct?

Reply:
i think so....i need junit 4 for the @ignore annotation.

Reply:
add <scope>test</scope>